### PR TITLE
[layouts][needs-docs] Replace layout icons

### PR DESCRIPTION
**Pull request by nyalldawson**
The previous layout icon (a blank page) was too difficult to distinguish from actions relating to pages themselves.

### CODE REVIEW

**Comment by nyalldawson** [comment]
Screenshots:

icons3
icon2
icons1

**Comment by nirvn** [comment]
Beyond the clear visual improvement, it's a nice UX improvement too for the layout designer toolbar (i.e. differentiating the "new page" action vs "new layout", "duplicate layouts", etc. actions).

IMHO ready to merge once the SVGs are compressed.

**Review by nirvn — APPROVED** [review]
On the basis of our discussions in google hangouts, I approve :wink:

**Comment by nyalldawson** [comment]
Here's the new report icon

report_icons

**Comment by NathanW2** [comment]
Nice.

**Comment by axelande** [comment]
Sweet.

File: src/ui/qgsoptionsbase.ui [hunk 1]
```diff
@@ -5530,6 +5530,34 @@ The bigger the number, the faster zooming with the mouse wheel will be.</string>
  </tabstops>
  <resources>
   <include location="../../images/images.qrc"/>
+  <include location="../../images/images.qrc"/>
```
Comment (jef-n) [inline src/ui/qgsoptionsbase.ui]: must be an important include ;)

**Pull request merged**